src/webui.c

```diff
@@ -56,6 +56,35 @@ static struct event_base *webui_base;
 static struct evhttp *webui_httpd;
 static struct evhttp_bound_socket *webui_listener;
 
+static void
+send_json_workers(struct evhttp_request *req, void *arg)
+{
+    struct evbuffer *buf = evhttp_request_get_output_buffer(req);
+    struct evkeyvalq *hdrs_in = NULL;
+    struct evkeyvalq *hdrs_out = NULL;
+    char rig_list[MAX_RIG_ID*MAX_RIGS+(MAX_RIGS*3)] = {0};
```

Comment:
This needs to be bigger. I suggest 1 MB and pass the start and end pointers to `account_rl`.